frontend/src/bundles/friends/components/friend-card/friend-card.tsx

```diff
@@ -8,41 +8,56 @@ import {
 import { ComponentSize } from '~/bundles/common/enums/enums.js';
 import { validateImageUrl } from '~/bundles/common/helpers/helpers.js';
 import { useCallback } from '~/bundles/common/hooks/hooks.js';
+import { type UserFollowingsResponseDto } from '~/bundles/friends/types/types.js';
 
 type FriendProperties = {
-    id: number;
-    name: string;
-    avatarUrl: string;
     isActive: boolean;
     isFollowed: boolean;
-    onToggleFollow: (id: number, isFollowed: boolean) => void;
-    message: (id: number) => void;
+    isCardSelected: boolean;
+    user: UserFollowingsResponseDto;
+    selectCard: (user: UserFollowingsResponseDto | null) => void;
+    onToggleFollow: (id: number) => void;
+    // message: (id: number) => void;
 };
 
 const FriendCard = ({
-    id,
-    name,
-    avatarUrl,
     isActive,
     isFollowed,
+    isCardSelected,
+    user,
+    selectCard,
     onToggleFollow,
-    message,
+    // message,
 }: FriendProperties): JSX.Element => {
+    const { id, fullName, email, avatarUrl } = user;
     const handleOnToggleFollow = useCallback(() => {
-        onToggleFollow(id, isFollowed);
-    }, [onToggleFollow, isFollowed, id]);
+        if (isCardSelected) {
+            selectCard(null);
+        }
+        onToggleFollow(id);
+    }, [onToggleFollow, id, isCardSelected, selectCard]);
 
-    const handleSendMessage = useCallback(() => {
-        message(id);
-    }, [message, id]);
+    const handleSelectCard = useCallback((): void => {
+        selectCard(user);
+    }, [selectCard, user]);
+
+    // const handleSendMessage = useCallback(() => {
+    //     messageFriend(id);
+    // }, [messageFriend, id]);
```

Comment:
Do we need this code here and all the comments in the frontend?

Reply:
I think you're right, I deleted it.